### PR TITLE
feat: Bump to Node v14

### PR DESCRIPTION
**Pull request by mgagliardo91**
## Why?

Allow repositories that use this SDK to move up in the node version.

## What changed?

- [x] Did you update the CHANGELOG?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,8 +29,8 @@
   "license": "ISC",
   "repository": "github:ndustrialio/contxt-sdk-js",
   "engines": {
-    "npm": "^5.10.0 ^6.0.0",
-    "node": ">=6"
+    "npm": "^5.6.0",
```
Comment (greg5green) [inline package.json:32]: Why disallow npm 6 and 7?

**Review by greg5green — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (mgagliardo91) [reply package.json:32]: ah, I meant >, but also, why do we need to restrict on `npm` if `node` is specified?

**Review by branchvincent — COMMENTED** [review]
looks good to me, but will defer to @greg5green

**Review by greg5green — APPROVED** [review]
perfect

**Pull request merged**